src/libsync/configfile.cpp

```diff
@@ -433,10 +432,17 @@ chrono::milliseconds ConfigFile::remotePollInterval(const QString &connection) c
     QSettings settings(configFile(), QSettings::IniFormat);
     settings.beginGroup(con);
 
-    auto defaultPollInterval = chrono::milliseconds(DEFAULT_REMOTE_POLL_INTERVAL);
+    auto defaultPollInterval { DefaultRemotePollInterval };
```

Comment:
no need for that var

Reply:
Hm, please check the code again. I think it is.

Reply:
ups